lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-ansible_tower_job.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Ansible_ConfigurationManager_AnsibleTowerJob < MiqAeServiceOrchestrationStack
```

Comment:
@bzwei 
The first Ansible should be AnsibleTower and the last part of the name should be Job.

We need to expose the class methods create_stack and instance method raw_status thats what the Automate methods would be calling